.bazelrc

```diff
@@ -99,7 +112,7 @@ build:nopenmp --
 # ~~~~ Memory handler (choose one) ~~~~
 
 # Build with tcmalloc
-build:tcmalloc --linkopt="-ltcmalloc"
+build:tcmalloc --linkopt -ltcmalloc
```

Comment:
Does this make any difference w/r to the previous format?

There are some `--copt arg` --> `--copt=arg` changes above, but this one goes the other way.

Reply:
I think the answer is no, and the inconsistency came about due to some back-and-forth debugging of a run command that took arguments and seemed to get parsed incorrectly. Still, the safest approach seems to be to use `=`, so in the most recent commit, I changed the ones that used spaces to use `=`.